code/cryptography/src/baconian_cipher/Baconian.java

```diff
@@ -35,35 +35,41 @@ public Baconian() {
     }
 
     public String encode(String word) {
-        StringBuilder results = new StringBuilder();
+        String results = "";
```

Comment:
I feel like the assignment isn't necessary.

Reply:
@arnavb  In Java variables must be initalized otherwise the compiler complains